src/tools/ScenarioMeasurement/Startup/ProcessTimeParser.cs

```diff
@@ -19,21 +19,21 @@ public void EnableUserProviders(TraceEventSession user)
         {
         }
 
-        public IList<Counter> Parse(string mergeTraceFile, string processName)
+        public IList<Counter> Parse(string mergeTraceFile, string commandLine)
         {
             var results = new List<double>();
             double threadTime = 0;
             var threadTimes = new List<double>();
             var ins = new Dictionary<int, double>();
             double start = -1;
             int? pid = null;
-            var imageFileName = "";
             using (var source = new ETWTraceEventSource(mergeTraceFile))
             {
 
                 source.Kernel.ProcessStart += evt =>
                 {
-                    if (evt.ProcessName.Equals(processName, StringComparison.OrdinalIgnoreCase))
+                    // comparing command line here to separate the test from set up and clean up commands with the same process name
+                    if (commandLine.Equals(evt.CommandLine.Replace("\"",""), StringComparison.OrdinalIgnoreCase))
```

Comment:
In commandLine value, ETW wraps the exe file with quotes so I have to remove it.

Reply:
You can probably simplify your logic here by getting the PID of the process under measurement when you start it and then save it for use during parsing.  That way, you don't have to worry about getting all of the escaping correct.